src/components/Dialog.tsx

```diff
@@ -163,6 +161,7 @@ export const Modal = styled.div<{ color: Color; constrain?: boolean }>`
   height: ${({ constrain }) => (constrain ? 'fit-content' : '100%')};
   left: 0;
   outline: ${({ theme }) => `1px solid ${theme.outline}`};
+  outline: transparent;
```

Comment:
the second outline value here seems wrong

Reply:
thx